config/foreman-answers.yaml

```diff
@@ -53,7 +68,16 @@ foreman::plugin::setup: false
 foreman::plugin::snapshot_management: false
 foreman::plugin::tasks: false
 foreman::plugin::templates: false
-foreman_proxy: {}
+
+## Smart Proxy and Plugins ##
+foreman_proxy:
+  puppet: false
+  puppetca: false
+  manage_puppet_group: false
+  ssl_ca: "/etc/foreman-pki/certs/ca/ca.crt"
+  ssl_cert: "/etc/foreman-pki/certs/smart-proxy/smart-proxy.crt"
+  ssl_key: "/etc/foreman-pki/certs/smart-proxy/smart-proxy.key"
+  ssldir: "/etc/foreman-pki/certs"
```

Comment:
This is wrong. ssldir is the Puppet CA SSL directory and only used to configure the Puppet CA cert provider. That's only used with Puppet 5, Puppet 6 uses Puppet's REST API.

You can argue the parameter has a bad name and you'd be right.

Reply:
You did a great job arguing the bad name for me :)

Reply:
I'm wondering what we should do. I hope to drop Puppet 5 support when it goes EOL in November. The REST API implementation also works with Puppet 5 but you miss some fields that are only shown in the UI so I think dropping that provider is fine. Then we can drop the parameter.

Given November is Foreman 2.3 or 2.4, I'm not sure renaming it for 2 or 3 releases is worth it.